src/main/java/author/AuthorList.java

```diff
@@ -36,4 +36,15 @@ public Author getAuthor(Author author) {
     public void addAuthor(Author author) {
         authorList.add(author);
     }
+
+    public int size() {
+        return authorList.size();
+    }
+
+    public void print() {
+        System.out.println("Here are the sla-I mean authors under you! Total: " + authorList.size());
+        for (int i = 0; i < authorList.size(); i++) {
+            System.out.println((i + 1) + ". " + authorList.get(i).getAuthorName());
```

Comment:
Nice! Possible to streamline this even further by having a `getAuthorName` at the `AuthorList` level?

Reply:
Sorry I don't understand. Are you talking about `authorList.get(i).getAuthorName()`?

Reply:
Yes, that's right.